### PR TITLE
Upgrade webpack to v4

### PR DESCRIPTION
**Pull request by darcien**
Webpack v4 have several breaking changes[1], and current version of `kfstart --type react` is broken.
And I kinda told Bandung to downgrade to webpack v3 till kfstart got fixed.
I tried to fix it, but I'm not confident enough with my fixes.
So, please review it if you're not busy 🙏 

[1] https://github.com/webpack/webpack/releases/tag/v4.0.0

### CODE REVIEW

**Comment by sstur** [comment]
Oops, I just saw this now. Can you coordinate with @taogilaaa to see if your code overlaps. May let him land his (since it's simpler) and then rebase this?

**Comment by darcien** [comment]
Ok, I already rebased. And there's no overlap, because @taogilaaa made changes to type `node` while mine is for type `react`.

File: src/main.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ function createProject(name, type) {
     let dstFile = configFiles[srcFile];
     sh.cp('-R', `${starterPath}/${srcFile}`, `./${dstFile}`);
   }
-  console.log('Creating direcotries ...');
+  console.log('Creating directories ...');
```
Comment (sstur) [inline src/main.js:116]: Wow, nice catch.

File: starter/react-web/webpack.config.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 /*eslint-env node */
 var path = require('path');
 
-var loaders = [
-  {test: /\.js$/, loader: 'babel-loader'},
+var rules = [
+  {
+    test: /\.js$/,
+    exclude: /node_modules/,
```
Comment (sstur) [inline starter/react-web/webpack.config.js:7]: Good fix.

**Review by sstur — APPROVED** [review]
Nailed it!

**Pull request merged**